composer.json

```
@@ -4,7 +4,7 @@
     "description": "PayPal Commerce Platform for WooCommerce",
     "license": "GPL-2.0",
     "require": {
-        "dhii/module-interface": "0.1",
+        "dhii/module-interface": "0.2.x-dev",
```

```suggestion
        "dhii/module-interface": "^0.2",
```

Maybe better to use this, that's what we do in other projects.

---

@AlexP11223 done, thanks for the info 👍